Walkthrough.md

```diff
@@ -62,6 +63,12 @@ For more information, check [Wallet FAQ](/FAQ/FAQ-Wallet.md)
 
 All of the invoices for your user account will be displayed here. You can filter the invoices by the status or a store. You can also create invoices manually. Invoices are sorted by date from new to old. You can open an individual invoice for more information.
 
+## Payment Requests
+
+Each store can have an unlimited number of payment requests which are displayed here. Payment requests are dynamic invoices that can be shared by url and paid at any time using current BTC exchange rates. Here you can edit, view and pay your payment requests. You can see invoice details for your payment requests and even clone previous payment requests for easy reoccuring requests for payment. 
```

Comment:
Too wordy and can be confused with subscription based payments, reformulate it somehow.

Reply:
> You can see invoice details for your payment requests and even clone previous payment requests for easy reoccuring requests for payment. 
Referring to this part

Reply:
Actually, after re-reading I think it's fine, sorry.

Reply:
yeah I want to fix it too.. I was trying to think of what the purpose of cloning is called :)

Reply:
Maybe:

> You can see invoice details for your payment requests and even clone previously created payment requests. 🤣